src/utils/datastructs.jl

```diff
@@ -93,6 +144,18 @@ struct SimpleExtremeValueDescr
    end
 end
 
+"""
+    SimpleExtremeValueDescr(data_type::String,
+                                 extremum::String,
+                                 peak_def::String)
+
+Defines a simple extreme day by its characteristics
+
+Input options:
+- data_type::String : Choose one of the attributes from the data you have loaded into ClustData
+- extremum::String : `min`,`max`
+- peak_def::String : `absolute`,`integral`
+"""
```

Comment:
Really great explanation of the different structs. It's also good to see that we can write the function in multiple lines. I thought that wasn't possible, but seems to work perfectly.